advanced/Templates/pihole-FTL.service.ini

```diff
@@ -0,0 +1,37 @@
+[Unit]
+Description=pihole-FTL Domain Name Server
+Documentation=man:pihole-FTL(8)
+After=remote-fs.target
+
+# Alternative based on https://wiki.debian.org/Hardening/Daemons#bind
+#   allows other services which rely on DNS to use pihole-FTL, but means
+#   breaking if config relies on remote-fs
+# After=network.target
+# Wants=nss-lookup.target
+# Before=nss-lookup.target
+
+After=resolvconf.service
+Wants=resolvconf.service
+# fail since dnsmasq is embedded
+Conflicts=dnsmasq.service
+
+[Service]
+# Run as unprivileged user, use AmbientCapabilities to assign capabilities
+User=pihole
+Group=pihole
+
+# start()
+ExecStartPre=+/bin/sh /etc/pihole/fix_files.sh
+ExecStartPre=+/bin/sh -c "echo \"nameserver 127.0.0.1\" | /sbin/resolvconf -a lo.piholeFTL"
+# Assign capabilities to the pihole user slice
+AmbientCapabilities=CAP_NET_BIND_SERVICE CAP_NET_RAW CAP_NET_ADMIN
+ExecStart=/usr/bin/pihole-FTL no-daemon
+
+## stop()
+# Clean up resolv.conf before shutting down service
+ExecStop=+/sbin/resolvconf -d lo.piholeFTL
+# Send SIGKILL 5 seconds after SIGTERM
+TimeoutStopSec=5
+
```

Comment:
```suggestion
ExecReload=/bin/kill -HUP $MAINPID
```

Reply:
Like with the timeout change, the init script currently does a full restart on `service pihole-FTL reload`, so it should probably involve a change to https://github.com/pi-hole/pi-hole/blob/master/advanced/Templates/pihole-FTL.service#L103 at the same time?

Reply:
Yes.